### PR TITLE
Patched various things for Linux

### PR DESCRIPTION
**Pull request by zachrammell**
The game now compiles, links, and runs without crashing.
(Tested on Debian Bullseye, gcc 9.2.1
There are graphics errors (most textures are black) but it runs smoothly.

### CODE REVIEW

File: src/Serialization.cpp [hunk 1]
```diff
@@ -153,8 +154,8 @@ void SerializeLevel(const char* level)
 std::vector<EntityID> DeserializeLevel(const char* level)
 {
   // Create a c-style string that represents the file path
+#ifdef _MSVC
```
Comment (HardGaming01) [inline src/Serialization.cpp:157]: Fixed this to compile in MSVC

File: CMakeLists.txt [hunk 2]
```diff
@@ -19,13 +19,26 @@ endif ()
 
 target_include_directories(uwuEngine PUBLIC ${FMOD_INCLUDE_DIR})
 target_include_directories(uwuEngine PUBLIC ${GLEW_INCLUDE_DIR})
+target_include_directories(uwuEngine PUBLIC lib/**/include)
```
Comment (HardGaming01) [inline CMakeLists.txt]: Not sure if this line is neccessary or not. There is a line above that should be doing the same thing.

**Review by HardGaming01 — APPROVED** [review]
Changed one pre processor condition to make it compile in MSVC, everything works perfectly

**Review by BrayanLopezDev — APPROVED** [review]
It works on my machine :)

**Pull request merged**